handbook/communication/code_of_conduct.md

```diff
@@ -0,0 +1,180 @@
+
+
+## Scope
+
+This code of conduct applies to any individual or community member that is representing the project or its community within both project spaces and public spaces, including (but not limited to) social media, online or offline events, and email or Slack exchanges.
+
+Note: Some Sourcegraph-managed communities have codes of conduct that pre-date this document and issue resolution process. While communities are not required to change their code, they are expected to use the resolution process outlined here. The review team will coordinate with the communities involved to address any concerns.
+
+
+## Enforcement
+
+We encourage all communities, teams, employees, and team members to resolve issues on their own whenever possible. This builds a broader and deeper understanding and ultimately a healthier interaction.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting conduct@sourcegraph.com. The conduct team is obligated to maintain confidentiality of the person reporting the incident at all times.
+
+In your report please include:
+
+-   Your contact information.
+-   Names (real, usernames or pseudonyms) of any individuals involved. If there are additional witnesses, please include them as well.
+-   Your account of what occurred, and if you believe the incident is ongoing. If there is a publicly available record (e.g., a mailing list archive or a public chat log), please include a link or screen captures.
+-   Any additional information that may be helpful.
+
+All reports will be reviewed by a multi-person team. Where additional perspectives are needed, the team may seek insight from others with relevant expertise or experience. Involved parties are never part of the review team.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately. If an individual engages in unacceptable behavior, the review team will take appropriate action, including a permanent ban from the community.
+
+
+# Conflict resolution
+
+_I disagree with the decision to…_
+
+_I disagree with the idea that…_
+
+Disagreements about decisions are a healthy part of team collaboration. By challenging each other individually, our team collectively discovers the best solution.
+
+
+## Disagreeing with a decision
+
+When you are challenging a decision, it is important to do so in a constructive way. Failure to do so may lead to [interpersonal conflict](https://about.sourcegraph.com/handbook/communication/conflict_resolution#interpersonal-conflict).
+
+Here is a process that you can follow to disagree with a decision productively.
+
+1. Proactively try to understand the decision by reviewing available information (Slack conversations, [RFCs](https://about.sourcegraph.com/handbook/communication/rfcs)).
```

Comment:
Use path references rather than urls for handbook links.

```suggestion
1. Proactively try to understand the decision by reviewing available information (Slack conversations, [RFCs](rfcs.md)).
```